ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Ledger/Query.hs

```diff
@@ -309,14 +309,17 @@ answerQueryAnytime HardForkLedgerConfig{..} =
 -------------------------------------------------------------------------------}
 
 data QueryHardFork xs result where
-  GetInterpreter :: QueryHardFork xs (History.Interpreter xs)
-  GetCurrentEra  :: QueryHardFork xs (EraIndex xs)
+  GetInterpreter  :: QueryHardFork xs (History.Interpreter xs)
+  GetCurrentEra   :: QueryHardFork xs (EraIndex xs)
+  GetLedgerCfg    :: QueryHardFork xs (HardForkLedgerConfig xs)
 
 deriving instance Show (QueryHardFork xs result)
 
 instance All SingleEraBlock xs => ShowQuery (QueryHardFork xs) where
-  showResult GetInterpreter = show
-  showResult GetCurrentEra  = show
+  showResult query = case query of
+    GetInterpreter  -> show
+    GetCurrentEra   -> show
+    GetLedgerCfg    -> const "TODO show `HardForkLedgerConfig xs`"
```

Comment:
Is this just for debugging? Is it ok to show a truncated version of the `HardForkLedgerConfig`?